tests/auto_bp_gossip_peering_test.py

```diff
@@ -84,7 +84,7 @@ def getHostName(nodeId):
     for nodeId in range(0, producerNodes):
         specificNodeopArgs[nodeId] = auto_bp_peer_arg
         producer_name = "defproducer" + chr(ord('a') + nodeId)
-        specificNodeopArgs[nodeId] += (f" --signature-provider {accounts[nodeId].activePublicKey}=KEY:{accounts[nodeId].activePrivateKey}")
+        specificNodeopArgs[nodeId] += (f" --signature-provider wire-{nodeId},wire,wire,{accounts[nodeId].activePublicKey},KEY:{accounts[nodeId].activePrivateKey}")
```

Comment:
Would be nice to for the first three values to be optional so previous syntax still worked.